src/main/java/com/carhouse/controller/WebController.java

```diff
@@ -0,0 +1,158 @@
+package com.carhouse.controller;
+
+import com.carhouse.model.*;
+import com.carhouse.model.stub.*;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class WebController {
+
+    @GetMapping("/homePage")
+    public String firstPage(Model model) {
+        List<CarMakeStub> listCarMake = new ArrayList<>();
+        listCarMake.add(new CarMakeStub(0, "Bentley", 3));
```

Comment:
Why is it here?

Reply:
It is used for a list of car makes  in the bottom of the page with a count of car sale with this car make